src/components/Input/Input.component.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+import {Icon} from '@Components/Icon';
+
+import {StyledInput, StyledFieldSet, StyledInputContainer} from './Input.styles';
+import {InputProps} from './Input';
+
+const DisplayFieldSet = (props: InputProps) => {
+    const {value, error, label} = props;
+
+    return (
+        <StyledFieldSet
+            className={`input_label ${value === '' ? '' : 'visible_label-border'} ${error ? 'input_error' : ''}`}
```

Comment:
@ayusht2810 Can't use `&&` as it returns `type = string|boolean` and className can be only `string` type.
@raghavaggarwal2308 can't use `??` as it returns LHS if LHS is true. we can't add input value in class name.

Created a function to extend class names instead of using the ternary operator.

Reply:
@tanisha-agarwal The nullish coalescing operator consider '' (empty string) as a truthy value. So, I guess you can use it here

Reply:
``` <legend className={value ?? 'visible_label'}>{label}</legend>```

Right @raghavaggarwal2308, but only when value is empty.
1. if `value` is empty then it will add `visible_label` to className
2. but if `value` is not empty then it will add `value` to className and  we can't show input value in class name.

Reply:
Oh okay got it, sorry my bad!